### PR TITLE
Italian App Store metadata

### PR DESCRIPTION
**Pull request by MrOgeid**
I noticed that the App Store metadata weren't translated in Italian. I hope that this can be useful.

### CODE REVIEW

**Comment by MrOgeid** [comment]
There's also new Italian translations for descriptions and chores. I added some missing strings too (I noticed that some of these are missing also in the German and French file.)
Fixed some typos too.

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -85,13 +85,13 @@ Più prezzi inserirai ogni giorno, dopo il lunedì mattina, migliori saranno le
...
 Se ti abbonerai potrai invece creare tutte le liste oggetti che desideri.";
-"ACHelperPlusPriceAndAboDetail %@ %@" = "Un abbonamento di %@  al mese sarà aggiunto al tuo account iTunes una volta data la conferma.
+"ACHelperPlusPriceAndAboDetail %@" = "Un abbonamento di %@ al mese sarà aggiunto al tuo account iTunes una volta data la conferma.
 Gli abbonamenti si rinnovano automaticamente se non vengono annullati 24 ore prima della loro scadenza.
 Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo account iTunes. I periodi di prova gratuita saranno annullati se acquisti un abbonamento.";
 "Subscribe for %@ / Month" = "Abbonati per %@ / Mese";
 "Buy lifetime AC Helper+ for %@" = "Acquista AC Helper+ per sempre per %@ ";
 "Thank you for your support!" = "Grazie mille per il tuo sostegno!";
-"%@ Monthly" = "%@ al mese";
+"%@ Monthly" = "%@ al mese";
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:94]:
```suggestion
"%@ Monthly" = "%@ al mese";
```

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -353,7 +361,7 @@ Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo acc
 "Squirrel" = "Scoiattolo";
 "Tiger" = "Tigre";
 "Wolf" = "Lupo";
-"Like" = "Gusti";
+"Like" = "Gusti";
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:364]:
```suggestion
"Like" = "Gusti";
```

**Comment by TheVaan** [comment]
You are right, there are some strings missing in French and German file. It’s difficult to track all changes because we all write in English and translation is a post process. I’m working on a translation service so we can track all strings that needs to be translated. Hope I’ll finish this in the next few days.

**Pull request merged**